bundles/net.tourbook/src/net/tourbook/preferences/PrefPageAppearance.java

```diff
@@ -315,6 +316,32 @@ private void createUI_40_OtherOptions(final Composite parent) {
          _chkShowInApp_MemoryMonitor.setText(Messages.pref_appearance_showMemoryMonitor);
 //         GridDataFactory.fillDefaults().indent(0, 10).applyTo(_chkMemMonitor);
       }
+      {
+         /*
+          * Duration of notification messages
+          */
+         final Composite notificationMessagesDurationContainer = new Composite(parent, SWT.NONE);
+         GridDataFactory.fillDefaults().span(2, 1).applyTo(notificationMessagesDurationContainer);
+         GridLayoutFactory.fillDefaults().numColumns(2).applyTo(notificationMessagesDurationContainer);
+         {
+            // label: duration
+            final Label lblNotificationMessagesDuration = UI.createLabel(notificationMessagesDurationContainer,
```

Comment:
I also created a template for it

![image](https://github.com/mytourbook/mytourbook/assets/1283445/c4413e3f-28e9-4d20-b505-158f4d459504)

Reply:
> I mostly use the simple names `container` and `label` when
> 
>     * Creating the UI
> 
>     * For local names
> 
>     * The names do not conflict within the variable scope

I usually inspire myself from the existing code as I assume that existing code respects the coding standards

In this case I used [this example from the same file](https://github.com/mytourbook/mytourbook/blob/67937a70b19b3e82437a3f6e0eb17b346b3e8237/bundles/net.tourbook/src/net/tourbook/preferences/PrefPageAppearance.java#L236)

Reply:
> Years ago, I also created a template for it
> 

I have never been able to use templates in Eclipse even after spending lots of time trying to configure them so i don't use them because of this reason

Reply:
> > I mostly use the simple names `container` and `label` when
> > ```
> > * Creating the UI
> > 
> > * For local names
> > 
> > * The names do not conflict within the variable scope
> > ```
> 
> I usually inspire myself from the existing code as I assume that existing code respects the coding standards
> 
> In this case I used [this example from the same file](https://github.com/mytourbook/mytourbook/blob/67937a70b19b3e82437a3f6e0eb17b346b3e8237/bundles/net.tourbook/src/net/tourbook/preferences/PrefPageAppearance.java#L236)

I used ` container ` until now 602 times 😄 

![image](https://github.com/mytourbook/mytourbook/assets/1283445/01fc7796-e39e-4fb4-99d9-38f5a13a3e44)